### ISSUE TITLE
[Dialogs] MDCAlertController does not announce 'Alert' in Voice Over mode on iOS 8.x

### ISSUE THREAD

**Issue by ianegordon** [material-components/material-components-ios#1644]
Reproducible in MDCCatalog on an iPhone 5 @ iOS 8.3:
Go to Dialogs -> AlertController
Enable Voice Over Mode
Select Show Alert

Expected behavior is for "Alert" to be spoken by VO.

This works as expected on iOS 10.x devices.

This functionality was originally added in #1639

**Comment by codeman7** [comment]
Infeasible
